### PR TITLE
Uep ov2020 2

### PR DESCRIPTION
**Pull request by mayavijx**
Updated the dockerfile with OV version 2020.2.

### CODE REVIEW

File: Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile [hunk 1]
```diff
@@ -42,25 +46,25 @@ RUN tar -xzf l_openvino_toolkit*.tgz && \
     ./install.sh -s silent.cfg && \
     cd - && \
     rm -rf l_openvino_toolkit* && \
-    cd /opt/intel/openvino/install_dependencies && ./_install_all_dependencies.sh && dpkg -i *.deb && \
-    pip install networkx==2.3 test-generator==0.1.1 defusedxml>=0.5.0
+    cd /opt/intel/openvino_2020.2.120/install_dependencies && ./_install_all_dependencies.sh && dpkg -i *.deb && \
+    pip3 install networkx==2.3 test-generator==0.1.1 defusedxml>=0.5.0
```
Comment (smkarlap) [inline Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile:50]: These are Model Optimizer dependencies and are no longer needed. This line should be removed.

File: Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile [hunk 2]
```diff
@@ -42,25 +46,25 @@ RUN tar -xzf l_openvino_toolkit*.tgz && \
...
 ENV LD_LIBRARY_PATH=${INTEL_OPENVINO_DIR}/deployment_tools/inference_engine/external/hddl/lib:$LD_LIBRARY_PATH
 RUN locale-gen en_US.UTF-8 && update-locale LANG=en_US.UTF-8
 ENV LANG en_US.UTF-8
 
-
 ARG DEVICE=CPU_FP32
-ARG ONNXRUNTIME_REPO=https://github.com/microsoft/onnxruntime
-ARG ONNXRUNTIME_BRANCH=master
+ARG ONNXRUNTIME_REPO=https://github.com/intel/onnxruntime
+ARG ONNXRUNTIME_BRANCH=openvino-ep-v2
```
Comment (smkarlap) [inline Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile:67]: Should point to 'master' in github.com/microsoft/onnxruntime as UEP is merged into it now.

File: Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile [hunk 3]
```diff
@@ -70,12 +74,12 @@ RUN wget https://github.com/Kitware/CMake/releases/download/v3.13.2/cmake-3.13.2
...
 
 RUN git clone --recursive -b $ONNXRUNTIME_BRANCH $ONNXRUNTIME_REPO && \
-    sed -i "s/sudo//" /onnxruntime/dockerfiles/scripts/install_common_deps.sh &&\
+   sed -i "s/sudo//" /onnxruntime/dockerfiles/scripts/install_common_deps.sh &&\
     /bin/sh /onnxruntime/dockerfiles/scripts/install_common_deps.sh &&\
-    cd /onnxruntime/cmake/external/onnx && python3 setup.py install 
+    cd /onnxruntime/cmake/external/onnx && python3 setup.py install
 
-RUN cd /onnxruntime && ./build.sh --config RelWithDebInfo --update --build --parallel --use_openvino $DEVICE --build_wheel
+RUN cd /onnxruntime && ./build.sh --config RelWithDebInfo --update --build --parallel --use_openvino $DEVICE --build_wheel --use_full_protobuf
```
Comment (smkarlap) [inline Azure-IoT-Edge/OnnxRuntime/UnifiedDocker/Dockerfile:81]: Build with config option 'Release' instead of 'RelWithDebInfo' to reduce container size.

**Comment by smkarlap** [comment]
Hi Maya, Is this ready for review again?

**Comment by mayavijx** [comment]
Hi Manohar,
It is ready for review now.
The changes made are:
1. Made config option for onnxruntime to Release.
2. Removed model optimizer dependencies.
3. Aggregate all RUN commands into one.
4. Removed the cmake 3.14 which is installed by onnxruntime/dockerfiles/scripts/install_common_deps.sh
5. Removed onnxruntime folder.

Now the image size is 2.07GB.

**Pull request merged**